core/utils/benchmark_main.cpp

```diff
@@ -67,7 +69,7 @@ namespace kagome {
 
     SL_INFO(logger, "Kagome benchmark stopped");
     logger->flush();
-
+    watchdog->stop();
```

Comment:
`gsl::finally([] { watchdog->stop(); })`?

Reply:
I don't see any particular reason for it now, but it won't hurt in the future so okay.